start.html

```diff
@@ -355,7 +556,114 @@ <h3 class="accordion">
+			 <p class="info_text" style="text-align:justify">
+			   The pbench-move-results script removes the
+			   results directory (assumed to be within the
+			   /var/lib/pbench-agent hierarchy) after
+			   copying it the results repo. But if there
+			   are previous results present (perhaps
+			   because pbench-move-results was never
+			   invoked, or perhaps because
+			   pbench-copy-results was invoked instead),
+			   pbench-move-results will copy <b>all</b>of
+			   them: you probably do not want that.
+			 </p>
+			 <p class="info_text" style="text-align:justify">
+			   It is a good idea in general to
+			   invoke <span class="c_snip_inline">pbench-clear-results</span>,
+			   which
+			   cleans <span class="c_snip_inline">/var/lib/pbench-agent</span>, <b>before</b>running
+			   your benchmark.
+                         </p>
+		       </div>
+		     </div>
+		   </div>
+		   <div class="accordionCard">
+		     <h3 class="accordion">
+		       <button class="accordionBtn pf-c-accordion__toggle" aria-expanded="true">
+			 <i class="arrowIcon fas fa-angle-right pf-c-accordion__toggle-icon" aria-hidden="true"></i>
+			 <span class="pf-c-accordion__toggle-text"> Kill tools</span>
+		       </button>
+		     </h3>
+		     <div class="explanation pf-c-accordion__expanded-content pf-m-fixed" hidden>
+		       <div class="pf-c-accordion__expanded-content-body"></div>
+		       <div class="pf-c-accordion__expanded-content-body">
+			 <p class="info_text" style="text-align:justify">
+			   If you interrupt a built-in benchmark
+			   script (or your own script perhaps), the
+			   collection tools are not going to be
+			   stopped. If you don't stop them explicitly,
+			   they can severely affect subsequent runs
+			   that you make. So it is strongly
+			   recommended that you
+			   invoke <span class="c_snip_inline">pbench-kill-tools</span>
+			   before you start your run:
+			 </p>
+			 <p class="c_snip">pbench-kill-tools --group=$group</p>
+			 <p class="info_text" style="text-align:justify">
+                           If you run Pbench from your own script, you should add a signal handler to do this:
+                         </p>
+			 <p class="c_snip">
+                           trap "pbench-kill-tools --group=$group" EXIT INT QUIT
+                         </p>
+		       </div>
+		     </div>
+		   </div>
+		   <div class="accordionCard">
+		     <h3 class="accordion">
+		       <button class="accordionBtn pf-c-accordion__toggle" aria-expanded="true">
+			 <i class="arrowIcon fas fa-angle-right pf-c-accordion__toggle-icon" aria-hidden="true"></i>
+			 <span class="pf-c-accordion__toggle-text"> Clear tools</span>
+		       </button>
+		     </h3>
+		     <div class="explanation pf-c-accordion__expanded-content pf-m-fixed" hidden>
+		       <div class="pf-c-accordion__expanded-content-body">
+			 <p class="info_text" style="text-align:justify">
+			   This tool will delete the tools.$group file
```

Comment:
Perhaps we should not mention what it does, but just that `pbench-clear-tools` will remove all registered tools by default from the default tool group.  You might want to also say that you can optionally remove specific tool groups that are registered, specific hosts that are registered, and specific tools that are registered.

Reply:
See issue #1927.